website/blog/2023-05-30-0.72-metro-package-exports-symlinks.md

```diff
@@ -0,0 +1,173 @@
+
+:::info
+Please update your config file to following [format](https://github.com/facebook/react-native/blob/76a42c292de838a0dd537935db792eaa81410b9b/packages/react-native/template/metro.config.js). Changes to `metro.config.js` will become required in 0.73. For 0.72, we will log a warning if not updated.
+:::
+
+This moves control over extending the base React Native Metro config into your project, and we’ve cleaned up the leftover defaults. In addition, this means that standalone Metro CLI commands, such as `metro get-dependencies`, will now work.
+
+#### Enabling pre-stable features
+
+To enable these new Metro features in your project, update your app’s `metro.config.js` file and set either the `resolver.unstable_enableSymlinks` or `resolver.unstable_enablePackageExports` options.
+
+```js
+const config = {
+  // ...
+  resolver: {
+    unstable_enableSymlinks: true,
+    unstable_enablePackageExports: true,
+  },
+};
+```
+
+### Developer Experience Improvements
+
+#### No more redboxes with invalid style properties
+
+Prior to this release, providing in an invalid style property in StyleSheet would result in a redbox. This is a high signal error that disrupts the developer workflow for a relatively low-risk error.
+
+In 0.72, we’ve relaxed this expectation to fail silently, like providing an invalid CSS property in the browser.
+
+#### Better error readability
+
+Hermes has added better error message when invoking an undefined callable.
+
+```
+    var x = undefined; x();
+    // Before: undefined is not a function
+    // After: x is not a function (it is undefined)
+```
+
+As well, LogBox stack traces now filter out internal Hermes bytecode frames that are not relevant to app users. [4c911a2dec](https://github.com/facebook/react-native/commit/4c911a2deceb59fc07735205ae3a4622b4334f88)
+
+#### Improved error output of the React Native CLI
+
+0.72 ships with React Native CLI v11.2.2 which includes improves to reduce duplication, clarify wording, reduce verbose stack traces, and add deep links to relevant docs in the following commands `init` , `run-android` and `run-ios`.
+
+You can find other improvements in the [React Native CLI changelogs](https://github.com/react-native-community/cli/releases/tag/v11.2.2).
+
+#### Faster compilation and JSON parsing in Hermes
+
+Hermes has improved the compilation time of large object literals. For example, in one reported issue, [#852](https://github.com/facebook/hermes/issues/852), a user had an entire dataset written out as a large object literal. By improving the de-duplication algorithm Hermes uses, compilation sped up by 97% ([221c](https://github.com/facebook/hermes/commit/221ce21a209e2e32a3eaaa2d9e28ca81842fad20)). These improvements will benefit build times for apps that bundle many objects.
```

Comment:
```suggestion
Hermes has improved the compilation time of large object literals. For example, in one reported issue, [#852](https://github.com/facebook/hermes/issues/852), a user had an entire dataset written out as a large object literal. By improving the de-duplication algorithm Hermes uses, compilation sped up by 97% ([see the commit](https://github.com/facebook/hermes/commit/221ce21a209e2e32a3eaaa2d9e28ca81842fad20)). These improvements will benefit build times for apps that bundle many objects.
```

Reply:
I'm going to keep it as the hash for now, as we use this format throughout the post